api/views.py

```diff
@@ -115,15 +117,37 @@ def get(self, request):
         return response.Response(data=data, status=status.HTTP_200_OK)
 
     def put(self, request):
+        self.updateImage(request)
         self.updateInterests(request)
         return self.update(request=request)
 
+    def updateImage(self, request):
+        imageData = None
+        try:
+            imageData = request.data.pop("image")
+        except KeyError:
+            # Image was not updated.
+            pass
+
+        if imageData is not None:
+            # Data is prepended by "data:image/*;base64,".
+            # Encode converts from string to bytes.
+            data_bytes = imageData.encode()
+
+            # Create the image content file.
+            created_at = now()
+            fname = "%d_%s" % (request.user.id, created_at)
```

Comment:
```suggestion
            fname = f"{request.user.id}_{created_at}"
```
Percent format string is fine but I suggest using f-strings. They are newer and arguably easier to read.